api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -176,7 +176,14 @@ message Bootstrap {
   }];
 
   // Optional watchdog configuration.
-  Watchdog watchdog = 8;
+  // This is for a single watchdog configuration for the entire system.
+  // Prefer to use *watchdogs* which has finer granularity.
+  Watchdog watchdog = 8 [deprecated = true];
+
+  // Optional watchdog configuration.
+  // This is used for specifying different watchdogs for the different subsystems.
+  // Only one of *watchdog* or *watchdogs* should be specified.
```

Comment:
nit: I would remove this, given the deprecation above, this can just fail in code with a good error message so we don't have to update the docs later.